### PR TITLE
Refine helm package and add installation docs

### PR DESCRIPTION
**Pull request by nkwangleiGIT**
Refine current helm package of arcadia and add installation docs.

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
+This document will introduce how to install Arcadia on a Kubernetes cluster.
+
+## Install and Run Arcadia
+
+#### Create or Use existing Kubernetes Cluster
```
Comment (bjwswang) [inline docs/installation.md:5]: should this be  `###` ?

File: docs/installation.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+```shell
+git clone https://github.com/kubeagi/arcadia.git
+``` 
+
+2. Get required images
+
+Container images are available in the Docker Hub. You can pull and push to your image registry or use them directly if your cluster can access docker hub.
+```
+# image of core controller
+docker pull kubeagi/arcadia-controller:v0.1.0
```
Comment (bjwswang) [inline docs/installation.md:28]: kubeagi/arcadia:v0.1.0

File: docs/installation.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+docker pull kubeagi/data-processing:v0.1.0
+# image of fastchat and its workers
+docker pull kubeagi/fastchat:v0.1.0
+docker pull kubeagi/fastchat-worker:v0.1.0
+# image of minio
+docker pull kubeagi/minio:RELEASE.2023-02-10T18-48-39Z
+# image of minio client to download objects
+docker pull kubeagi/minio-mc:RELEASE.2023-02-10T18-48-39Z
+# image of bff-server, contains apiserver and portal
+docker pull kubeagi/kubeagi-portal:v0.1.0
```
Comment (bjwswang) [inline docs/installation.md:39]: Currently the image has been pushed to `yuntibuild/kubeagi-portal`.

File: deploy/charts/arcadia/values.yaml [hunk 4]
```diff
@@ -41,13 +63,32 @@ fastchat:
...
+  image:
+    repository: kubeagi/minio
+    tag: RELEASE.2023-02-10T18-48-39Z
   mode: standalone
   rootUser: "admin"
   rootPassword: "Passw0rd!"
   persistence:
     enabled: true
-    storageClass: ""
+    storageClass: "openebs-hostpath"
```
Comment (bjwswang) [inline deploy/charts/arcadia/values.yaml:74]: Can we keep this empty by default?  Most probably user shall use the default storageclass.

File: deploy/charts/arcadia/templates/ingress-minio.yaml [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+  annotations:
+    description: minio-api-ingress
+    displayName: minio-api-ingress
+    httpSend: /
+    ingress-lb: portal-ingress
+    kubernetes.io/ingress.class: portal-ingress
+    nginx.ingress.kubernetes.io/enable-access-log: "false"
+    nginx.ingress.kubernetes.io/enable-rewrite-log: "false"
+    nginx.ingress.kubernetes.io/load-balance: round_robin
+    nginx.ingress.kubernetes.io/proxy-body-size: ""
```
Comment (bjwswang) [inline deploy/charts/arcadia/templates/ingress-minio.yaml:16]: Maybe set `nginx.ingress.kubernetes.io/proxy-body-size` to `1024m`

File: deploy/charts/arcadia/templates/kubebb-menus.yaml [hunk 6]
```diff
@@ -0,0 +1,357 @@
+apiVersion: core.kubebb.k8s.com.cn/v1alpha1
+kind: Menu
+metadata:
+  creationTimestamp: null
```
Comment (bjwswang) [inline deploy/charts/arcadia/templates/kubebb-menus.yaml:4]: should we move this `kubebb-menues.yaml` and `kubebb-portal.yaml` inside `portal.yaml` ?

File: deploy/charts/arcadia/templates/ingress-minio.yaml (hunk 5 above)
Comment (bjwswang) [reply deploy/charts/arcadia/templates/ingress-minio.yaml:16]: We need to upload large files in portal

File: deploy/charts/arcadia/templates/apiserver.yaml [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+              valueFrom:
+                fieldRef:
+                  apiVersion: v1
+                  fieldPath: metadata.namespace
+            - name: DEFAULT_CONFIG
+              value: {{ .Release.Name }}-config
+          command:
+            - "./graphql-server"
+          args:
+            - "--enable-playground={{ .Values.apiserver.enableplayground }}"
```
Comment (bjwswang) [inline deploy/charts/arcadia/templates/apiserver.yaml:38]: should we expose a `-v={{ .Values.apiserver.log_level}}` ?

File: deploy/charts/arcadia/templates/ingress-minio.yaml (hunk 5 above)
Comment (nkwangleiGIT) [reply deploy/charts/arcadia/templates/ingress-minio.yaml:16]: increased to 10240m = 10G

File: docs/installation.md (hunk 1 above)
Comment (nkwangleiGIT) [reply docs/installation.md:5]: that's on purpose to make it more smaller

File: docs/installation.md (hunk 2 above)
Comment (nkwangleiGIT) [reply docs/installation.md:28]: updated

File: deploy/charts/postgre-oci.yaml [hunk 8]
```diff
@@ -0,0 +1,23 @@
+apiVersion: core.kubebb.k8s.com.cn/v1alpha1
+kind: Repository
+metadata:
+  annotations:
+    kubectl.kubernetes.io/last-applied-configuration: |
```
[the remark this answers is not in the document]
Comment (nkwangleiGIT) [reply deploy/charts/postgre-oci.yaml:5]: mention this in the installation doc

File: deploy/charts/arcadia/templates/kubebb-menus.yaml (hunk 6 above)
Comment (nkwangleiGIT) [reply deploy/charts/arcadia/templates/kubebb-menus.yaml:5]: that's make it separate for now

File: deploy/charts/arcadia/templates/ingress-minio.yaml [hunk 9]
```diff
@@ -0,0 +1,77 @@
...
+    nginx.ingress.kubernetes.io/proxy-buffering: "on"
+    nginx.ingress.kubernetes.io/proxy-connect-timeout: "60"
+    nginx.ingress.kubernetes.io/server-alias: ""
+    nginx.ingress.kubernetes.io/ssl-redirect: "true"
```
[the remark this answers is not in the document]
Comment (nkwangleiGIT) [reply deploy/charts/arcadia/templates/ingress-minio.yaml:20]: update the datasource to fore it use HTTPs

**Comment by nkwangleiGIT** [comment]
Testcase might fail if arcadia-apiserver restart too many times before arcadia-minio datasource is created, as arcadia-minio creation is a post-install action for helm install, use separate issue to track.

**Pull request merged**